ros_ws/src/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+/opt/ros/kinetic/share/catkin/cmake/toplevel.cmake
```

Comment:
Add the symlink to the .gitignore as the given path is not universal for each operating system

Reply:
Okay i am going to fix this